_layouts/person.html

```diff
@@ -100,7 +100,30 @@
         </small>
     </dd>
     {% endif %}
-</dl>
+  </dl>
+
+</div>
 
+<div class="person__actions">
+
+  {% if person.last_party.image %}
+    <div class="person__party_emblem">
+      <img src="{{ person.last_party.proxy_image|fixproxyurl }}/240/0"/>
```

Comment:
Probably, I don't think that block will ever be reached since we don't set last_party or pull in the party data, right? I'll get rid of it for now.